### PR TITLE
Sass build refactor + add breadcrumbs and article tags

### PR DESCRIPTION
**Pull request by laras126**
Starting to actually build patterns and configure class defaults for modules. This is the first of probably several PRs that fill in the functionality gaps and get everything to fit together.

* Moved the lightweight Sass build into the base larva/ repo so it can be run on all packages
* Added utilities and algorithms
* Started configuring breadcrumbs and article tags with default CSS classes
* Improved webfont handling with custom properties

Documentation is happening in some disparate places at the moment, but this section is particularly relevant to this PR / Sass compilation: https://github.com/penske-media-corp/pmc-larva/tree/feature/asset-build-refactor/packages/css-algorithms#naming-the-scss-file

### CODE REVIEW

File: packages/larva/lib/utils/renderSass.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const sass = require( 'node-sass' );
+const path = require( 'path' );
+
+module.exports = function renderSass( sassData ) {
+	return new Promise( ( resolve, reject ) => {
+		sass.render( {
+			data: sassData,
+			includePaths: [
+				path.resolve( '/Users/laraschenck/Sites/pmc/pmc-larva/packages/css-utilities/node_modules') // command should be run from same directory as node_modules
```
Comment (escobar022) [inline packages/larva/lib/utils/renderSass.js]: Dynamically resolve local path

File: packages/patterns/components/c-link/c-link.twig [hunk 2]
```diff
@@ -1,3 +1,3 @@
-<a class="c-link {{ modifier_class }} {{ c_link_classes }}" href="{{ c_link_url }}">
+<a class="c-link {{ modifier_class }} // {{ c_link_classes }}" href="{{ c_link_url }}">
```
Comment (escobar022) [inline packages/patterns/components/c-link/c-link.twig]: Is this commented out?

File: packages/larva/lib/utils/renderSass.js (hunk 1 above)
Comment (laras126) [reply packages/larva/lib/utils/renderSass.js]: Ah, that is a place holder – I have a scaffold for a function to get the path that I need to fill in still.

File: packages/patterns/components/c-link/c-link.twig (hunk 2 above)
Comment (laras126) [reply packages/patterns/components/c-link/c-link.twig]: No - that's a text separator for between the actual pattern's name and utilities, though it does look like a comment which I hadn't thought of, and now that I'm looking at it, I think that should only be on modules. It's intended to help with readability in the DOM e.g. 
Screen Shot 2019-07-30 at 5 39 29 PM

**Pull request merged**